modules/querier/config.go

```diff
@@ -25,13 +25,10 @@ func (cfg *Config) RegisterFlagsAndApplyDefaults(prefix string, f *flag.FlagSet)
 		MatchMaxConcurrency:   true,
 		MaxConcurrentRequests: cfg.MaxConcurrentQueries,
 		Parallelism:           2,
-		GRPCClientConfig: grpcclient.ConfigWithTLS{
-			GRPC: grpcclient.Config{
-				MaxRecvMsgSize:     100 << 20,
-				MaxSendMsgSize:     16 << 20,
-				UseGzipCompression: false,
-				GRPCCompression:    "gzip",
-			},
+		GRPCClientConfig: grpcclient.Config{
```

Comment:
the previous settings are weird:

```
  UseGzipCompression: false,	
  GRPCCompression:    "gzip",
```

Does that mean we were or were not using gzip?  Does this PR change that?

Reply:
Here is the code in question:  https://github.com/cortexproject/cortex/blob/v1.6.0/pkg/util/grpcclient/grpcclient.go#L67

Afaict, we were using gzip.   `UseGzipCompression:true` would override the compression algorithm, but a `false` would not.